src/context.js

```diff
@@ -42,16 +44,33 @@ class Context {
   }
 
   async init () {
+    if (this.pollUrl) {
```

Comment:
write method `function isEnteringFromAuthRequest() { return this.pollUrl != null; }`